src/server/game/World/World.cpp

```diff
@@ -452,6 +452,10 @@ void World::LoadConfigSettings(bool reload)
     rate_values[RATE_XP_BG_KILL]  = sConfigMgr->GetFloatDefault("Rate.XP.BattlegroundKill", 1.0f);
     rate_values[RATE_XP_QUEST]    = sConfigMgr->GetFloatDefault("Rate.XP.Quest", 1.0f);
     rate_values[RATE_XP_EXPLORE]  = sConfigMgr->GetFloatDefault("Rate.XP.Explore", 1.0f);
+
+    m_int_configs[CONFIG_XP_BOOST_DAYMASK] = sConfigMgr->GetIntDefault("XP.Boost.Daymask", 0);
+    rate_values[RATE_XP_BOOST] = sConfigMgr->GetFloatDefault("XP.Boost.Rate", 1.0f);
```

Comment:
Seems I forgot to change the default rate from 1 to 2 here as it was changed in config.
```suggestion
    rate_values[RATE_XP_BOOST] = sConfigMgr->GetFloatDefault("XP.Boost.Rate", 2.0f);
```

Reply:
Nice suggest feature.

Reply:
Shouldn't that magic number taken directly from config? So now if needed rate 3.0, it is needed to edit in 2 places?

Reply:
I think you have misread the code (although I could be wrong).
`1.0f` / `2.0f` is the default value, while `XP.Boost.Rate` reads the config value.
I presume the default is included in case the config value is missing, out of scope or invalid.